### PR TITLE
Add countries and send email info

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

File: content/user/running/login_as_customer.md [hunk 1]
```diff
@@ -57,3 +57,10 @@ In older Zen Cart versions a plugin is required in order to login as a customer.
 - Encrypted Master Password
 
 These plugins are all available in the [Plugins Library](https://www.zen-cart.com/downloads.php). 
+
+
+## What if the user doesn't have an account? 
+
+- If they have an email address, you can sipmly create an account for them in the storefront using their email address and a randomly chosen password
```
Comment (drbyte) [inline content/user/running/login_as_customer.md:64]: `sipmly`

File: content/user/admin_pages/locations/countries.md [hunk 2]
```diff
@@ -4,6 +4,15 @@ category: admin_pages
...
 ---
 
-Defines every country and ISO-3166 2-letter and 3-letter country code. Also defines the address format for letters sent by postal mail.
+Defines every country with the following information: 
 
-For more information, see [address formats](/user/localization/address_formats/). 
+- Country Name
+- ISO-3166 2-letter and 3-letter country code 
+- [Address format](/user/localization/address_formats/)
+- Whether you ship to that country (since 1.5.7)
```
Comment (drbyte) [inline content/user/admin_pages/locations/countries.md:12]: country `status` was added v1.5.2

**Review by drbyte — COMMENTED** [review]
A couple typos.

**Comment by lat9** [comment]
In the section on resetting a customer's password, should it be noted that HTML special characters can't be used?  Otherwise, the password's not going to be seen as a match on the storefront (since the storefront doesn't sanitize those inputs but the admin does).

**Comment by scottcwilson** [comment]
This sounds like a bug - can you write up an issue with repro steps for on https://github.com/zencart/zencart/issues ?

**Comment by drbyte** [comment]
> In the section on resetting a customer's password, should it be noted that HTML special characters can't be used? Otherwise, the password's not going to be seen as a match on the storefront (since the storefront doesn't sanitize those inputs but the admin does).

Fixed in https://github.com/zencart/zencart/commit/821f07c5ae460705f570a0375b14ecd658eb087f

**Comment by lat9** [comment]
It's still, IMO, worth documenting the restriction for sites running zc156 through zc157.

**Pull request merged**